index.js

```diff
@@ -233,7 +253,18 @@ class SalaryPaycheck {
   }
 }
 
-export {
+/**
+ * Round down a number to the specified decimal places
+ *
+ * @param {number} value Amount to be truncated
+ * @param {number} [places] Decimal places to truncate
+ */
+const truncateNumber = (value, places = 2) => {
```

Comment:
could just use `.toFixed(2)` not to create too much own logic?

Reply:
if we were outputting to string, i would, but we need to have them as numbers

Reply:
maybe just `+(...).toFixed(2)` ? 
adding logic for doing standard operations is overkill here, IMHO.

Reply:
changed, but this will be rounding the number either way instead of just down as before.